test/integration/cloud-config-hwc.yml.template

```diff
@@ -0,0 +1,10 @@
+# This is the configuration template for ansible-test HWC integration tests.
```

Comment:
This file is not needed since there is no cloud test plugin using it.

Reply:
thanks, I will remove it!